includes/class-img-converter.php

```
@@ -12,795 +12,799 @@
 
-							$avif_path = $this->get_img_path( $source_image, 'avif' );
-							Util::prepare_cache_dir( dirname( $avif_path ) );
+						try {
 
-							if ( imageavif( $image, $avif_path, $quality ) ) {
+							if ( file_exists( $webp_path ) ) {
 								$this->update_conversion_status( $source_image, 'completed', $format );
+								return true;
+							}
+							// Initialize Imagick and read the image file.
+							$imagick = new \Imagick();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '242', column '23'). (undefined)

(MissingImport)